bumpversion.sh

```diff
@@ -10,10 +10,13 @@ bumpversion_build() {
 
 bumpversion_release() {
   bump2version patch
-  VERSION=$(bump2version --list --tag --commit --allow-dirty release | grep -oP '^new_version=\K.*$')
-  git push origin $RELEASE_BRANCH
+  git config user.name "aodn-ci-build"
+  git config user.email "a.mckeown@utas.edu.au"
```

Comment:
Email needs to be changed. Wish we could give it a name referring to bump version rather than ci-build, given this isn't necessarily run on ci-build...

Reply:
Done @akashisama